tasks/containers/TaskWorker.js

```diff
@@ -79,14 +79,32 @@ export default compose(
 
```

Comment:
@NotThatSmith in case you aren't already aware (i wasn't for a while): as a general debugging tip for this situation, i recommend using `debugger` (add a line with just `debugger`). it's like `console.log('value', value)`, but better since it halts execution and you can see all the local variables at the time. 🔍  then you can play it for every time it reaches the questionable moment and see what is happening there.